README.md

```diff
@@ -89,10 +89,12 @@ Options:
   -t, --tap [path]            Output results in tap with optional file path
   -x, --junit [path]          Output results in junit xml with optional file path
   -o, --timeout <length>      Set timeout for npm install
-  -f, --fail-flaky            Ignore flaky flags. Don't ignore any failures.
+  -f, --fail-flaky            Ignore flaky flags. Do not ignore any failures.
```

Comment:
please send this as a different PR if you would like. It will need to be updated in the yargs options as well